aws-auth-cognito/src/main/java/com/amplifyframework/statemachine/StateMachine.kt

```diff
@@ -101,17 +136,30 @@ internal open class StateMachine<StateType : State, EnvironmentType : Environmen
         }
     }
 
+    /**
+     * Unregister a listener.
+     * @param token token of the listener to remove
+     */
     private fun removeSubscription(token: StateChangeListenerToken) {
         pendingCancellations.remove(token)
         subscribers.remove(token)
     }
 
+    /**
+     * Sends `event` to the StateMachine for resolution, and applies any effects and new states returned from the resolution.
```

Comment:
Can you do a pass for verb tense here?  Like how this function `sends` an event but the next one below `notif(ies)` all the listeners, etc.  Do we have a style guide for Javadoc-type comments?

Reply:
Not sure if we have style guide, but I have updated the verb tense.